cpp/include/cudf/column/column_view.hpp

```diff
@@ -104,6 +104,11 @@ class column_view_base {
    *---------------------------------------------------------------------------**/
   size_type size() const noexcept { return _size; }
 
+  /**---------------------------------------------------------------------------*
+   * @brief Returns true if `size()` returns zero, or false otherwise
+   *---------------------------------------------------------------------------**/
+  size_type empty() const noexcept { return size() == 0; }
```

Comment:
I would highly recommend that we call this method `is_empty`. It is a widely acknowledged mistake that `empty` was the wrong name for the predicate method. It is unclear whether `empty` means `clear` (as in to empty the contents) or whether it is asking whether the data structure is empty (with either `true` or `false` response). All recent C++ features opt for the name `is_empty`, see:
https://en.cppreference.com/w/cpp/types/is_empty
https://en.cppreference.com/w/cpp/filesystem/is_empty

I even asked about this on twitter after listening to episode 73 of cppcast, and the recommendation was to always but `is_` or `has_` in front of predicates: https://twitter.com/hatcat01/status/1007161606994845697?s=20

Reply:
Thanks Conor, that's great info. Will change it.

Reply:
Done.